external/CMakeLists.txt

```diff
@@ -34,6 +34,25 @@ target_include_directories(css-color-parser-cpp
   PUBLIC
   ${CMAKE_CURRENT_SOURCE_DIR}/css-color-parser-cpp)
 
+## zlib ##
+if (CMAKE_CXX_COMPILER_ID MATCHES "MSVC")
```

Comment:
In many of these statements, the conditional should not be whether the compiler is "MSVC", but whether the target platform is Windows. You could compile tangram-es for Windows using CLion or Cygwin and the compiler would be "Clang" or "GCC" but we would still need to build and link these libraries.